lib/json_api_client/associations/belongs_to.rb

```diff
@@ -11,14 +11,20 @@ def belongs_to(attr_name, options = {})
       end
 
       class Association < BaseAssociation
+        include Helpers::URI
         def param
           :"#{attr_name}_id"
         end
 
         def to_prefix_path
           "#{attr_name.to_s.pluralize}/%{#{param}}"
         end
+
+        def set_prefix_path(attrs)
+          attrs.merge!(param => encode_part(attrs[param])) if attrs.key?(param)
```

Comment:
This will modify the input hash which *could* have issues in the future. It looks like it's currently receiving a copy (modified) from some other input, but we should rebind `attrs = attrs.merge(...) if ...`

Alternatively, if we want to keep modifying the hash, just set the value rather than merging in another hash of one item: `attrs[param] = encode_part(attrs[param]) if attrs.key?(param)`